projects/README.md

```diff
@@ -0,0 +1,19 @@
+The 'projects' directory contains all the different projects added by the contributors.
+
+Following directory structurs is followed:
```

Comment:
Don't you think this change will be quite like a duplication from contribution.md

Reply:
Agreed. Since I needed some text to add the README file, I put this text. I can remove it and only have a link to the contributing.md file.

Reply:
Updated the text.